src/mots/config.py

```diff
@@ -107,6 +107,67 @@ def write(self, hashes: dict | None = None):
             yaml.dump(self.config, f)
 
 
+def check_nested_keys_for_value(
+    dictionary: dict, keys: list | tuple, boolean=True
+) -> bool:
+    """Check a nested key for a value. If boolean is False, checks if the key exists.
+
+    For example:
+        >>> test = {"a": {"b": {"c": 1, "d": None}}}
+        >>> check_nested_keys_for_value(test, ["a", "b", "c"])
+        True
+        >>> check_nested_keys_for_value(test, ["a", "b", "e"])
+        False
+        >>> check_nested_keys_for_value(test, ["a", "b", "c", "d"])
+        False
+    """
+    value = dictionary
+    for key in keys:
+        if value and isinstance(value, dict):
+            value = value.get(key)
+        else:
+            return False
+    if boolean:
+        return bool(value)
+    else:
+        return True
+
+
+def reference_anchor_for_module(
+    index: int,
+    person: dict,
+    key: str,
+    file_config: FileConfig,
+    directory: Directory,
+    module: dict,
+) -> None:
+    """Associate person with a reference to a directory entry if possible.
+
+        :param index: the position of the person in the referrer field
+        :param person: a dictionary with information about the person to be added
+        :param key: the key (e.g, "peers" or "owners_emeritus") that defines context
+        :param file_config: config context being modified
+        :param module: dictionary of the module in the config being modified
+
+
+    This is needed so that ruamel.yaml can correctly reference anchors to people.
+    """
+    logger.debug(f"Setting reference to {person} as {key} in {module['machine_name']}")
+
+    referrer = module["meta"][key] if key.endswith("_emeritus") else module[key]
```

Comment:
Will the `meta` key always exist?

Reply:
Yes, if folks are following the correct convention of putting the `emeritus` keys in meta. If they don't, other things should fail before this but this will all be fixed when we have better yaml validation (which is on my TODO list here: https://bugzilla.mozilla.org/show_bug.cgi?id=1753716).